dockerfiles/platform-build-incremental-container/Dockerfile

```diff
@@ -0,0 +1,47 @@
+##############################################################################
+#
+#    HPCC SYSTEMS software Copyright (C) 2021 HPCC Systems®.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+##############################################################################
+
+# Base container image that builds all HPCC platform components
+
+ARG CR_USER=hpccsystems
+ARG CR_REPO=docker.io
+ARG PLATFORM_PRBASE_VER
+FROM ${CR_REPO}/${CR_USER}/platform-build:${PLATFORM_PRBASE_VER}
+
+USER hpcc
+
+WORKDIR /hpcc-dev/HPCC-Platform
+ARG GITHUB_REPO
+RUN git remote add upstream https://github.com/${GITHUB_REPO}
+RUN git fetch upstream +${GITHUB_PRREF}
+RUN git checkout FETCH_HEAD
+RUN git submodule update --init --recursive
+
+WORKDIR /hpcc-dev/build
+
+ARG BUILD_THREADS
+RUN if [ -n "${BUILD_THREADS}" ] ; then echo ${BUILD_THREADS} > ~/build_threads; else echo $(nproc) > ~/build_threads ; fi
```

Comment:
I think 
```
ARG BUILD_TYPE=RelWithDEbInfo
ARG USE_CPPUNIT=1
RUN cmake <PARAMS> /hpcc-dev/HPCC-Platform
'''
or similar lines are missing to build platform

Reply:
not needed, cmake doesn't need to be rerun explicitly.
cmake will rerun if make deems it necessary (if e.g. CMakeLists.txt files etc have changed), and the existing BUILD_TYPE and USE_CPPUNIT settings (from base that's already been built using them) will be reused.